public/pages/CreateMonitor/containers/CreateMonitor/CreateMonitor.js

```diff
@@ -127,6 +131,15 @@ export default class CreateMonitor extends Component {
     else this.onCreate(monitor, formikBag);
   }
 
+  backendErrorHandler(actionName, data) {
+    toastNotifications.addDanger({
+      title: `Failed to ${actionName} the monitor`,
+      text: data.resp,
+      toastLifeTimeMs: 30000, // the default is 10 sec
```

Comment:
Could you please double check this? During testing, I remember seeing this message forever, till I clicked on [X]

Reply:
Sure, I will double check it tomorrow. The toasts for backend errors are expected to appear for 30 seconds, and we can adjust the time if needed.

Reply:
Hi, after checking, I can see the toast message dismisses after 30 sec, and should we shorten the duration?

Reply:
Thanks for checking. 30secs seems long, lets change it to 20sec

Reply:
Got it.